README.md

```diff
@@ -1,3 +1,108 @@
+
+<img src="https://i.imgur.com/wMlXUuO.png" width="800"/>
+
+In the pop-up window set a topic name and display name for your success alarm and hit the `create topic` button. 
+
+<img src="https://i.imgur.com/QtmsevM.png" width="800"/>
+
+You will be navigated to the topic details page for your new SNS Topic.
+Click the `create subscription` button.
+
+<img src="https://i.imgur.com/x7t8yUT.png" width="800"/>
+
+Here you can choose the type of notification you want, for this example, we will set up an email notification for this alarm.
+Place your email address in the `Endpoint` field and hit `Create subscription`.
+
+<img src="https://i.imgur.com/C13ikXf.png" width="800"/>
+
+Check your email inbox and confirm the subscription to your SNS alarm. 
+
+<img src="https://i.imgur.com/2Wksf2l.png" width="800"/>
+
+Once you have successfully confirmed your subscription in your email, you should see it listed in the `Subscriptions` table on the `Topic details` page for your SNS Alarm.
+
+<img src="https://i.imgur.com/W6SAfep.png" width="800"/>
+
+You will need to repeat this process for both of your success and failure alarms.
+On each of the `Topic details` pages for both of your alarms, you need to copy the `Topic ARN` value and paste it into their respective parameter fields in your Data Pipeline setup.
+
+#### Schedule your Job
+
+<img src="https://i.imgur.com/C9R631G.png" width="800"/>
+
+Once your ARN parameters are set for your alarms you can move on to scheduling when this pipeline will run.
+Set the cadence for when the job will run and make sure you set your starting and ending dates to something in the future.
+You can also set the job to run only on pipeline activation if you would rather manually start your job.
+
+#### Finish The Pipeline Setup
+
+Once you have your parameters set and scheduled all you need to do is set an S3 bucket location for the logs from the runs and you can click `Activate` at the bottom of the page.
+This will run a check on all of your settings and confirm that everything should work. 
+Somethings you might run into here could be setting improper values for the instance types for your clusters (Note: not all EC2 types work in EMR and Data Pipeline), or your schedule settings don't make sense. 
+
+<img src="https://i.imgur.com/NV3OIlc.png" width="800"/>
+
+#### Test Your Job
+
+<img src="https://i.imgur.com/UQ5w5oE.png" width="800"/>
+
+Once your job activates successfully you should be done!
+If you are doing a test run first, just wait for the time that the pipeline is scheduled to start and it should run and email you on the result of the run.
+If you want to view the console/standard output of your job you can find it in the `Emr Step Logs` link and click the `stdout.gz` link to view/download the output.
+Here you will also find the error output if something goes wrong in `stderr.gz` or the logs from the pipeline itself in the `controller.gz` if you need to do some debugging with your pipeline. 
```

Comment:
I would split this second part into a new sentence. "The logs from the pipeline itself are in the controller.gz file."